### ISSUE TITLE
starting netdata on boot with init.d on Debian Jessie

### ISSUE THREAD

**Issue by ovizii** [firehol/netdata#1347]
Followed the instructions but when it came to this step:

```
update-rc.d netdata defaults
insserv: Service named has to be enabled to start service netdata
insserv: Service apache2 has to be enabled to start service netdata
insserv: Service httpd has to be enabled to start service netdata
insserv: Service squid has to be enabled to start service netdata
insserv: Service nginx has to be enabled to start service netdata
insserv: Service mysql has to be enabled to start service netdata
insserv: Service opensips has to be enabled to start service netdata
insserv: Service upsd has to be enabled to start service netdata
insserv: Service hostapd has to be enabled to start service netdata
insserv: Service lm_sensors has to be enabled to start service netdata
insserv: exiting now!
update-rc.d: error: insserv rejected the script header
```

**Comment by ktsaou** [comment]
@lhw @FedericoCeratto any ideas on this?

**Comment by FedericoCeratto** [comment]
@ovizii where did you get the initscript from? Can you link to a copy please?

**Comment by ovizii** [comment]
Sorry for the lack of info, let me rectify that. Netdata running happily along for a while now, then I noticed an update and clicked the link to the instructions and landed here: https://github.com/firehol/netdata/wiki/Installation 

so here I go:
```
cd /tmp
git clone https://github.com/firehol/netdata.git --depth=1
cd netdata
./netdata-installer.sh
<ENTER>
```

Then I see the area below: starting netdata at boot 
I skipped to the init.d section
```
cp system/netdata-lsb /etc/init.d/netdata
chmod +x /etc/init.d/netdata
update-rc.d netdata defaults
```
which resulted in above error.

**oh btw this time I see:**
Update script generated   : ./netdata-updater.sh 
does that mean I don't have ot go throug hteh whole installation again and just run the udpater script?

**No action is necessary however as I realized I can use: service netdata start|stop|restart**
